### ISSUE TITLE
The new jar-in-jar mclib-20.jar dependency is breaking eclipse ide

### ISSUE THREAD

**Issue by vicmatskiv** [bernie-g/geckolib#531]
I'm migrating my mod from 1.20.1/4.2.3 to 1.20.4/4.3.1 version. It looks like the newly introduced jar-in-jar dependency mclib-20.jar is not working in eclipse, at least genEclipseRuns task cannot produce a proper eclipse project classpath configuration.
The resulting eclipse project shows the following error:
```The project was not built since its build path is incomplete. Cannot find the class file for com.eliotlash.mclib.math.IValue. Fix the build path then try building this project	forge-1.20.4-49.0.13-mdk		Unknown	Java Problem```.

I tried forge versions 49.0.13 and 49.0.14.

When I add an explicit dependency on mclib, the game fails to start with: 
```java.lang.module.ResolutionException: Module forge reads more than one module named mclib.```

Launching the game from command line with `./gradlew runClient` works fine.

**Comment by vicmatskiv** [comment]
Getting the same issue with Visual Studio Code:

The type com.eliotlash.mclib.math.IValue cannot be resolved. It is indirectly referenced from required type software.bernie.geckolib.core.animation.AnimationController

**Comment by Tslat** [comment]
Is your explicit dependency compileonly? or implementation
Try compileonly

**Comment by vicmatskiv** [comment]
Tried that as well:
```
compileOnly(group: 'com.eliotlash.mclib', name: 'mclib', version: "${mclib_version}")
```

still getting an error:

```
Caused by: java.lang.module.ResolutionException: Module forge reads more than one module named mclib
```

**Comment by Tslat** [comment]
Come to the discord and show your build.gradle

**Comment by Tslat** [comment]
Will be fixed in 1.21

For the time being, Eclipse is just broken
